### ISSUE TITLE
Doesn't seem to work in Mailable

### ISSUE THREAD

**Issue by ibrentlam** [TerrePorter/StringBladeCompiler#21]
Hello, 
In an ordinary controller, this works fine, but when I call the exact same code in a Mailable class, it doesn't do the substitution, as:
 $results = view(['template' => $body], ['unsubscribe' => 'click this link to unsubscribe']);
Any ideas?
Thanks

**Comment by TerrePorter** [comment]
Hi, sorry just noticed the mail. Family medical emergency this week has the schedule all messed up.

I'm not sure it $result should be the generated template.  I'm assuming that you have confirmed the template contents is valid. Did you dd the $result and see what it creates?

**Comment by ibrentlam** [comment]
Yes, I tried the exact line of code in a regular controller and it worked
fine.
The result had the template, but with the {{ $unsubscribe }} wasn't
interpolated.

It's nothing time-critical, take the time you need for the family
emergency. Family comes first. Thanks,
Brent

**Comment by TerrePorter** [comment]
@ibrentlam Sorry it has taken me so long to get a chance to back to this. Did you get it figured out ?

**Comment by ibrentlam** [comment]
Terre,

No, I haven't figured it out. Your help would be greatly appreciated, as
I'll be needing it in several other places.

Brent

**Comment by TerrePorter** [comment]
@ibrentlam The way they have the mailer set it checks for a string name of a view so it fails when trying to send a full stringview object or even an array. 

However, I was able to get this to work.

```
/**
     * Build the message.
     *
     * @return $this
     */
    public function build()
    {
        return $this->view(
            [
                'html' => ['template' => ' {{ $token }} html version'],
                'text' => ['template' => ' {{ $token }} plain version']
            ],
            [
               'token' => 'test 1 : '. time()
            ]
        );
    }
```

It should also work even if your not passing the template data as part of the view statement. 

Give it a shot and let me know how it goes.

**Comment by TerrePorter** [comment]
@ibrentlam just checking to see if that worked for you ?

**Comment by ibrentlam** [comment]
Thanks,

Haven't gotten a chance to try it out. Hopefully will early next week. I'll
keep you posted, Thanks

brent

**Comment by dimsanchuk** [comment]
@TerrePorter I tried this and it works.

Offtop: Why are there still not released version 3.4?

**Comment by TerrePorter** [comment]
@Dimsanchuk Glad it worked. I've released a version for 3.4. I was just behind in everything.

**Comment by dimsanchuk** [comment]
Thanks.
